test/integration/targets/vmware_guest_disk_controller/tasks/main.yml

```diff
@@ -0,0 +1,80 @@
+# Test code for the vmware_guest_disk_controller module
+# Copyright: (c) 2019, Diane Wang (Tomorrow9) <dianew@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- when: vcsim is not defined
+  block:
+  - import_role:
+      name: prepare_vmware_tests
+    vars:
+      setup_attach_host: true
+      setup_datastore: true
+      setup_virtualmachines: true
+
+  - name: gather disk controllers facts of the virtual machine
+    vmware_guest_disk_controller:
+      validate_certs: False
+      hostname: "{{ vcenter_hostname }}"
+      username: "{{ vcenter_username }}"
+      password: "{{ vcenter_password }}"
+      name: "{{ virtual_machines[0].name }}"
```

Comment:
I get: `"missing required arguments: datacenter"`

Reply:
Is it the right datacenter "dc1" I can use? Thanks.

Reply:
Yes, `'{{ dc1 }}'`